### PR TITLE
UMH-3458: Add Breadcrumb component

### PR DESCRIPTION
**Pull request by jomadoye**
http://localhost:3000/openmrs-standalone/owa/openmrs-owa-labworkflow/index.html#/FakeBreadcrumbPage
screen shot 2018-08-30 at 11 41 51 am

http://localhost:3000/openmrs-standalone/owa/openmrs-owa-labworkflow/index.html#/
screen shot 2018-08-30 at 11 41 59 am

### CODE REVIEW

File: app/js/components/shared/BreadCrumb/BreadCrumb.jsx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+                  className="glyphicon glyphicon-chevron-right breadcrumb-item separator"
+                  aria-hidden="true" />
+                <span className="title breadcrumb-item">
+                  <strong>Lab workflow</strong>
+                </span>
+              </Link>
+            )
+        }
+        {
+          FakeBreadcrumbPage
```
Comment (jomadoye) [inline app/js/components/shared/BreadCrumb/BreadCrumb.jsx]: Adding this here so everyone has an example of how to create a breadCrumb

File: package.json [hunk 2]
```diff
@@ -71,7 +78,7 @@
     "build:dev": "npm run clean && NODE_ENV=\"development\" webpack --progress --colors",
     "build:prod": "npm run test && npm run build",
     "build:deploy": "NODE_ENV=\"deploy\" webpack --progress --colors --target=web",
-    "watch": "NODE_ENV=\"development\" webpack --progress --colors --watch --cache=false",
+    "watch": "NODE_ENV=\"deploy\" webpack --progress --colors --watch --cache=false",
```
Comment (jomadoye) [inline package.json]: Added this change to fix an issue with the live reload

File: app/js/components/shared/BreadCrumb/BreadCrumb.jsx [hunk 3]
```diff
@@ -0,0 +1,100 @@
+/*
```
Comment (jomadoye) [inline app/js/components/shared/BreadCrumb/BreadCrumb.jsx]: This component is fully managed by its state, it watches for changes in the URL Path and updates it states accordingly. It's mostly static and I can see it getting quite complex if we start having deeply nested pages e.tc
For now the styling is also minimal.

File: .gitignore [hunk 4]
```diff
@@ -2,3 +2,4 @@ node_modules
 bower_components
 dist
 config.json
+npm-debug.log**
```
Comment (rhenshaw56) [inline .gitignore]: you should also add `*.zip` to prevent pushing of the generated zip file

**Comment by jomadoye** [comment]
@mogoodrich I agree with snapshot test, especially for components that are purely presentational and components like this that tend to change often, 
I have also updated the PR to your suggestion. That's definitely a cleaner way of rendering the breadcrumb.

File: .gitignore (hunk 4 above)
Comment (jomadoye) [reply .gitignore]: Added @rhenshaw56

**Pull request merged**